src/command_modules/azure-cli-acr/README.rst

```diff
@@ -9,54 +9,55 @@ Commands to manage Azure container registries
         az acr: Commands to manage Azure container registries.
 
     Subgroups:
-        credential: Manage admin user credential for Azure container registries.
+        credential: Manage administrator login credentials for Azure container registries.
         repository: Manage repositories for Azure container registries.
 
     Commands:
-        check-name: Check whether the container registry name is available.
-        create    : Create a container registry.
-        delete    : Delete a container registry.
-        list      : List container registries.
-        show      : Get a container registry.
-        update    : Update a container registry.
+        check-name: Checks whether the container registry name is available for use.
+        create    : Creates or updates a container registry with the specified parameters.
```

Comment:
While I recognize the SDK call is "create_or_update", it will confuse users as to why you have a command called create that "creates or updates" and a command called update that just updates--and why they update differently. I would keep the help the same as it was--i.e. that it just creates (and you can then drop the 'with the specified parameters' part because that is implied in all commands).